backend/config/config.go

```diff
@@ -60,6 +64,10 @@ func Load(cfgFile *string) (*Config, error) {
 		return nil, fmt.Errorf("failed to post process config: %w", err)
 	}
 
+	if err = c.Validate(); err != nil {
+		log.Fatalf("failed to validate config: %s", err)
```

Comment:
I think it is better to return the error, because the `Load` function can return an error and every checks above also return an error.